### PR TITLE
Maybe mention living WHO therapeutics guidelines

### PR DESCRIPTION
**Pull request by dziakj1**
I'm not a co-author on this one but I thought it might be good for you to also mention the living guidelines article at #991 which would be interesting to readers of this article.
  

### Related issues

#991

### Suggested reviewers (optional)

@rando2

### Checklist

- [x] Text is formatted so that each sentence is on its own line.

- [x] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/20.pharmaceuticals.md [hunk 1]
```diff
@@ -105,6 +105,7 @@ Combinations of interventions, such as hydroxychloroquine with azithromycin, are
 The purpose of this review is to provide an evolving resource tracking the status of efforts to repurpose and develop drugs for the treatment of COVID-19.
 We highlight four strategies that provide different paradigms for the identification of potential pharmaceutical treatments.
 As results become available from additional clinical trials, we will continue to update this manuscript to keep pace with the current understanding of which therapeutics may be effective against SARS-CoV-2 or COVID-19.
+Summary information can also be found in the World Health Organization's living guidelines document for COVID-19 therapeutics
```
Comment (agitter) [inline content/20.pharmaceuticals.md:108]:
```suggestion
The WHO guidelines [@doi:10.1136/bmj.m3379] and a systematic review [@doi:10.1136/bmj.m2980] are complementary living documents that summarize COVID-19 therapeutics.
```

**Review by agitter — CHANGES_REQUESTED** [review]
I agree we should reference this related project as well as the connected systematic review.  I added both citations in my suggestion.

**Review by rando2 — APPROVED** [review]

**Comment by rando2** [comment]
@agitter is it OK with you if I merge this?

**Comment by agitter** [comment]
> is it OK with you if I merge this?

Yes, looks good to me

**Pull request merged**